packages/primitives/src/Menu.tsx

```diff
@@ -0,0 +1,178 @@
+/**
+ * Copyright (c) 2024-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { menuAnatomy } from "@ark-ui/anatomy";
+import { Menu } from "@ark-ui/react";
+import { cva, sva } from "@ndla/styled-system/css";
+import { styled } from "@ndla/styled-system/jsx";
+import { StyledVariantProps, SystemStyleObject } from "@ndla/styled-system/types";
+import { createStyleContext } from "./createStyleContext";
+import { Text, TextProps } from "./Text";
+
+const itemStyle: SystemStyleObject = {
+  display: "flex",
+  alignItems: "center",
+  borderRadius: "xsmall",
+  cursor: "pointer",
+  textStyle: "label.medium",
+  paddingBlock: "4xsmall",
+  paddingInline: "xsmall",
+  gap: "xsmall",
+  transitionDuration: "fast",
+  transitionProperty: "background, color",
+  transitionTimingFunction: "default",
+  outline: "none",
+  _hover: {
+    textDecoration: "underline",
+  },
+  _highlighted: {
+    textDecoration: "underline",
+  },
+  _disabled: {
+    color: "text.disabled",
+    cursor: "not-allowed",
+    "& svg": {
+      color: "stroke.disabled",
+    },
+    _hover: {
+      color: "text.disabled",
+      textDecoration: "none",
+      background: "surface.default",
+      "& svg": {
+        color: "stroke.disabled",
+      },
```

Comment:
Trenger color-valgene oppgis på nytt for `_disabled_hover` når det ikke er noen forskjell fra `_disabled`? Er det noe Pandamagi som jeg misser her som gjør at det må eksplisitt oppgis "dobbelt"?

Reply:
Ikke noe panda-magi, men css-greier! En disablet knapp kan fortsatt hovres. Det vil vi ikke ha noe av, så vi overstyrer hover-verdiene også.

Reply:
Var koblingen mellom `_disabled` og `data-disabled` / `aria-disabled` jeg ikke var klar over (og som jeg da ville klassifisert som "pandamagi", forstår det som en fordel med nettopp Panda at det er sånn (selv om det ikke nødvendigvis er unikt)). Og da forstår jeg at de må oppgis en ekstra gang også her - takk for kjapt svar!

Reply:
Jeg ville ikke klassifisert dette som panda-magi, altså. Dette gjelder også helt vanlige knapper med helt vanlig css.
https://codepen.io/Jonas-Carlsen/pen/mdYBmVE

Reply:
Du misforstår; jeg skjønner naturligvis at knapper kan hoveres uavhengig av om de er disablet eller ikke, og veit det er basis CSS. Oppdatert slik, så vil du se hva jeg mener, den blå knappen her vil ikke bli rød ved hover fordi disabled allerede har en farge:
![image](https://github.com/NDLANO/frontend-packages/assets/33912044/6f47deb8-d5e3-4e3b-855d-22ad67241b22)

Uansett irrelevant, jeg fikk svaret jeg trengte om hvorfor det må stå flere ganger. Takk igjen :)